Makefile

```diff
@@ -160,6 +160,21 @@ test/functional:
 	# Run the functional tests against an existing cluster. Make sure you have logged in to the cluster.
 	go clean -testcache && go test -v ./test/functional -timeout=80m
 
+.PHONY: test/products/local
+test/products/local:
+	# Running the products tests against an existing cluster inside a container. Make sure you have logged in to the cluster.
+	# Using 'test-containers.yaml' as config and 'test-results' as output dir
+	mkdir "test-results"
+	$(CONTAINER_ENGINE) pull quay.io/integreatly/delorean-cli:master
+	$(CONTAINER_ENGINE) run --rm -e KUBECONFIG=/kube.config -v "${HOME}/.kube/config":/kube.config:z -v $(shell pwd)/test-containers.yaml:/test-containers.yaml -v $(shell pwd)/test-results:/test-results quay.io/integreatly/delorean-cli:master delorean tests run --test-config ./test-containers.yaml --output /test-results --namespace test-products
+
+.PHONY: test/products
+test/products:
+	# Running the products tests against an existing cluster. Make sure you have logged in to the cluster.
+	# Using 'test-containers.yaml' as config and 'test-results' as output dir
+	mkdir "test-results"
+	delorean tests run --test-config ./test-containers.yaml --output ./test-results --namespace test-products
```

Comment:
when we run this inside CI, I don't think we should create the directory here. Let's make the output dir configurable, so that we can control the output directory in the pipeline.

Reply:
@wei-lee updated